doc-content/enterprise-only/admin-and-config/exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -39,6 +44,28 @@ dashbuilder/navigation/readme.md
 VERSION
 ----
 
+
+If you want to export specific dashbuilder data as a ZIP file, do the following tasks:
```

Comment:
I would mention that in the **Custom export**, only the user created data can be exported (in the export all option, there are also business central internal data sets, that are exported)

Note: at this moment, also the Custom export contains the BC internal data sets, but that is a bug and should be fixed in the CR2 build of the 7.8.0